index.js

```diff
@@ -0,0 +1,15 @@
+const hamburger = document.querySelector('.hamburger');
```

Comment:
Original image            |  Your image
:-------------------------:|:-------------------------:
<img width="1440" alt="Screenshot 2023-01-03 at 18 29 21" src="https://user-images.githubusercontent.com/69011963/210410456-296a85d1-0e33-4ae6-9d85-e8a33c8c9a36.png"> | <img src="https://user-images.githubusercontent.com/69011963/210410443-7af5c3a3-b70a-4c29-bfca-a9ee6f1aa010.png" width="1440"/>

- [ ] As requested, please ensure the design of the mobile menu adheres to the initial design, including font, colours, images, text, and margins. Please amend accordingly.